man/subscription-manager.8

```diff
@@ -1013,16 +983,12 @@ Overall Status: Insufficient
 .RE
 
 .SS DEPRECATED COMMANDS
-As the structures of subscription configuration have changed, some of the original management commands have become obsolete. These commands have been replaced with updated commands.
+As the structures of subscription configuration have changed, some of the original management commands have become obsolete. Some of these commands have been replaced with updated commands.
```

Comment:
ditto, this will be covered by #3447

Reply:
This is resolved by commit: [c42fd22](https://github.com/candlepin/subscription-manager/pull/3444/commits/c42fd2264ec7b4607c469a307232589cebc7b075)